CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+## v0.8.0 - 2021-05-25
+
+We're happy to announce the release of Lokomotive v0.8.0 (Hogwarts Express).
+
+### Changes in v0.8.0
+
+#### Kubernetes updates
+
+- Update AKS to `1.18.17` ([#1466](https://github.com/kinvolk/lokomotive/pull/1466)).
+
+#### Component updates
+
+- Update `prometheus-operator` to `0.46.0` ([#1440](https://github.com/kinvolk/lokomotive/pull/1440)).
+- Update `contour` to `v1.13.1` ([#1450](https://github.com/kinvolk/lokomotive/pull/1450)).
+- Update `calico` to `v3.18.1` ([#1453](https://github.com/kinvolk/lokomotive/pull/1453)).
+
+#### Terraform provider updates
+
+- Update Terraform providers to their latest versions ([#1451](https://github.com/kinvolk/lokomotive/pull/1451)).
+
+#### Features
+
+- Add a certificate rotation command: `lokoctl cluster certificate rotate` ([#1435](https://github.com/kinvolk/lokomotive/pull/1435)).
+- Add `reclaim_policy` field to components `rook-ceph`, `openebs-storage-class` and `aws-ebs-csi-driver`. Change the default behaviour of the default storage class to `Retain` from `Delete`. ([#1369](https://github.com/kinvolk/lokomotive/pull/1369)).
+
+#### Deprecation and Removal
+
+- Remove the `webhook` field from the `cert-manager` component ([#1413](https://github.com/kinvolk/lokomotive/pull/1413)).
+
+#### Configuration syntax changes
```

Comment:
This is usually under `Updating from vX.X.X`.

Reply:
Done.